docs/modules/ROOT/pages/integrations/database.adoc

```diff
@@ -0,0 +1,69 @@
+= Database Integrations
+
+== Overview
+
+KubeArchive needs a database to store the resources to make them retrievable.
+KubeArchive currently supports integration with PostgreSQL.
+MariaDB integration is in progress.
+
+[NOTE]
+====
+
+KubeArchive integration test suite runs with PostgreSQL 16.6
+
+====
+
+== Configuration and Customization
+
+KubeArchives offers a [filename]`kubearchive.sql` file with the DDL to create
+the schema as part of the installation.
+
+[IMPORTANT]
+====
+The provided schema includes indexes.
+
+KubeArchive users should review and customize the indexes
+if needed based on the expected queries and the amount and
+the distribution of the expected archived data.
+====
+
+Specific instructions for the configuration of the PostgreSQL Database are available in
+xref:/getting-started/instllation.adoc#_install_and_configure_kubearchive[Install and Configure KubeArchive].
+
+== Support new databases
+
+KubeArchive database integration is designed to include new databases.
+
+To add a new database integration the following interfaces need to be implemented:
+
+* link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/facade/creator.go[DBCreator]
+* link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/facade/deleter.go[DBDeleter]
+* link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/facade/filter.go[DBFilter]
+* link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/facade/inserter.go[DBInserter]
+* link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/facade/selector.go[DBSelector]
+* link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/facade/sorter.go[DBSorter]
+
+[NOTE]
+====
+Except for the `DBCreator` functions, all the functions in the interfaces receive/output objects of the
+link:https://pkg.go.dev/github.com/huandu/go-sqlbuilder[sqlbuilder Go library].
+Check out the docs to see how to work with them.
```

Comment:
```suggestion
All the interfaces, with the exception of `DBCreator`, work with objects from the
link:https://pkg.go.dev/github.com/huandu/go-sqlbuilder[sqlbuilder Go library].
Check out the docs to see how to work with them.
```

Reply:
The sentence was incorrect, I change the instructions so now there is no "except" in the sentence.